.eslintrc

```diff
@@ -83,6 +83,7 @@
           "to"
         ]
       }
-    ]
+    ],
+    "linebreak-style": 0
   }
 }
```

Comment:
Add newline, please. 

To make your life easier, check if your editor has the newline setting and enable it. For example, I'm using Visual Studio Code editor and I have the following in the settings:
```
{
  "files.insertFinalNewline": true,
  "files.trimFinalNewlines": true
}
```
I don't have to worry about it as the editor automatically adds it, and delete extra newlines at the end of file